spread/google.go

```diff
@@ -433,6 +433,15 @@ func (p *googleProvider) createMachine(ctx context.Context, system *System) (*go
 		diskParams["diskSizeGb"] = int(system.Storage / gb)
 	}
 
+	secureBootParams := googleParams{}
+	if system.SecureBoot {
+		secureBootParams = googleParams{
```

Comment:
... and this immediately reallocates it. It's not a big deal, but avoiding it is also trivial.